manifests/0000_50_olm_13-operatorgroup-default.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
```

Comment:
i'm not sure why this one was being excluded either, i don't think it should be.